internal/backend/storage/fs/rcs.go

```diff
@@ -62,14 +62,17 @@ func (s *Store) Revisions(context.Context, string) ([]backend.Revision, error) {
 		}}, nil
 }
 
-// GetRevision is not implemented.
-func (s *Store) GetRevision(context.Context, string, string) ([]byte, error) {
-	return []byte("foo\nbar"), nil
```

Comment:
Nice catch xD

Reply:
I put that in during early development. And never removed it :(